<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ImportCsvDialog</class>
 <widget class="QDialog" name="ImportCsvDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Import from CSV</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../ico/appicon.png</normaloff>../ico/appicon.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="spacing">
        <number>2</number>
       </property>
       <item>
        <widget class="QLabel" name="label_separator">
         <property name="text">
          <string>Separator</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboBox_separator">
         <property name="frame">
          <bool>true</bool>
         </property>
         <item>
          <property name="text">
           <string>,</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>tab</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>;</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>-</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>/</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLineEdit" name="lineEdit_tableName">
     <property name="acceptDrops">
      <bool>false</bool>
     </property>
     <property name="placeholderText">
      <string>Name of new deck</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPlainTextEdit" name="plainTextEdit">
     <property name="focusPolicy">
      <enum>Qt::ClickFocus</enum>
     </property>
     <property name="placeholderText">
      <string>Vocabulary,Definition,Pronunciation(optional)</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="label_numLines">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ImportCsvDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>244</x>
     <y>506</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ImportCsvDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>312</x>
     <y>506</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
